tools/validator.py

```diff
@@ -42,7 +42,9 @@ def CheckFile(self, filename):
           result = False
 
     except errors.FormatError as exception:
-      logging.warning(exception.message)
+      logging.warning(
+          u'Unable to validate file: {0:s} with error: {1:s}'.format(
```

Comment:
I think it's ugly. Since you almost always want a string conversion just being explicit when it *isn't* a string makes sense to me, but leave it if you like.